client/webserver/site/src/js/markets.js

```diff
@@ -1710,7 +1710,7 @@ export default class MarketsPage extends BasePage {
     const manager = new OrderTableRowManager(tr, orderBin, base.info.unitinfo, rateConversionFactor)
     tr.manager = manager
     bind(tr, 'click', () => {
-      this.reportDepthClick(tr.manager.getRate())
+      this.reportDepthClick(tr.manager.getRate() / rateConversionFactor)
```

Comment:
Just like the `mouseenter` handler below.  Now I can click on either chart or table and it reports the click rate correctly everywhere.